lib/java/com/google/android/material/bottomnavigation/BottomNavigationItemView.java

```diff
@@ -126,7 +126,11 @@ public void initialize(MenuItemImpl itemData, int menuType) {
     if (!TextUtils.isEmpty(itemData.getContentDescription())) {
       setContentDescription(itemData.getContentDescription());
     }
-    TooltipCompat.setTooltipText(this, itemData.getTooltipText());
```

Comment:
I'm unsure how `itemData.getTooltipText()` works on pre-API 26, because [it was only added in that version](https://developer.android.com/reference/android/view/MenuItem#getTooltipText()).

I had to set Robolectric config to 26 to avoid a `NoSuchMethodException`.